processing/data_collection/gazette/spiders/rs_bento_goncalves.py

```diff
@@ -0,0 +1,62 @@
+from dateparser import parse
+import datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RsBentoGoncalvesSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4302105"
+    name = "rs_bento_goncalves"
+    allowed_domains = ["bentogoncalves.rs.gov.br"]
+    start_urls = ["http://www.bentogoncalves.rs.gov.br/diario-oficial"]
+    start_date = dt.date(2014, 9, 19)
+
+    def parse(self, response):
+        for gazette in self.parse_gazettes(response):
+            yield gazette
+
+        pagesURLSelector = '//*[@id="conteudo"]/div[2]/ul/div[2]/div/a/@href'
+        pages = response.xpath(pagesURLSelector).getall()
+
+        for page in pages:
+            url = 'http://www.' + self.allowed_domains[0] + '/' + page
+            yield scrapy.Request(url, self.parse_gazettes)
+
+    def parse_gazettes(self, response):
+        gazetteLinksSelector = '//*[@id="conteudo"]/div[2]/ul/li/p/a[contains(@class, \'txt-130p\')]/@href'
+        gazetteTitleSelector = '//*[@id="conteudo"]/div[2]/ul/li/p/a/strong/text()'
+
+        gazetteLinks= response.xpath(gazetteLinksSelector).getall()
+        gazetteTitles = response.xpath(gazetteTitleSelector).getall()
+
+        for i in range(0, len(gazetteTitles)):
```

Comment:
If you are iterating over the titles and use the index `i` just to access the titles, you can access them directly. 
```suggestion
        for title in gazetteTitles):
```

Reply:
I used `i` to iterate over `gazetteLinks` and `gazetteTitles`, since they have the same length, but I can 
zip them like `for title, link in zip(gazetteTitles, gazetteLinks)`

Reply:
Oh... I missed that use. Sorry. I would say that the idea of using zip seems to be nice. If you want to added, go ahead.